lib/modules/hosts/strawpoll.js

```diff
@@ -0,0 +1,11 @@
+addLibrary('mediaHosts', 'strawpoll', {
+	domains: ['strawpoll.me'],
```

Comment:
Can add `attribution: false,` here (Strawpoll adds its own "Powered by strawpoll.me" link at the bottom)